_includes/footer.html

```diff
@@ -7,8 +7,8 @@ <h2 class="footer-heading">{{ site.title }}</h2>
     <div class="footer-col-wrapper">
       <div class="footer-col footer-col-1">
         <ul class="contact-list">
-          <li>{{ site.title }}</li>
-          <li><a href="mailto:{{ site.email }}">{{ site.email }}</a></li>
+          <li><a href="mailto:{{ site.ian_email }}">Contact Ian</a></li>
+          <li><a href="mailto:{{ site.matt_email }}">Contact Matt</a></li>
```

Comment:
Thoughts on changing to  "Email Ian" and "Email Matt"? I'm not sure what to expect when I see a link that says "Contact"